content/posts/android-tv-12.md

```diff
@@ -0,0 +1,142 @@
+</div>
+
+## Updated user interface design
+
+After logging in you will be presented with a modern look and feel in our refreshed user interface. A new toolbar is added to the top-right corner of the screen to open the search page, settings or to switch to a different user. No need to scroll down to the bottom of the home screen anymore.
+Beneath this new toolbar is your media, like it always was, but the cards got a new look with the debatable colored backgrounds removed.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-home.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-home.png" />
+</div>
+
+The settings screen got a refresh too. It allows you to more easily change settings. Some new settings got added to customize the app to your own tastes.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-settings.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-settings.png" />
+</div>
+
+The user interfacer when browsing inside libraries got some slight changes too. We're hoping to completely revamp this part of the app at some point, but that doesn't stop us from making it slightly better right now.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-browse.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-browse.png" />
+</div>
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-details.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-details.png" />
+</div>
+
+### Themes
+
+The web client supported theming since forever but the Android TV app did not. That changes today with the introduction of three themes: Muted Purple together with Dark (Default) and Classic Emerald are now available in the app to supports basic theming. We're planning to expand this feature in the future with more themes and more variety between them. _Maybe we should add a light theme?_
+
+### New video player design
+
+The video player was completely redesigned to remove a lot of clutter and make it easier to use. The new design is bigger so you can more easily see the information on a television.
+
+<div class="juxtapose">
+    <img data-label="0.11" src="/images/posts/androidtv-0-12-0/11-player.png" />
+    <img data-label="0.12" src="/images/posts/androidtv-0-12-0/12-player.png" />
+</div>
+
+The new video player design is just the beginning. A project to rewrite the playback code behind it is already in the works. This rewrite should help with the crashes and unnecessary transcoding that happens sometimes. It will also give us the opportunity to add new features like SyncPlay. But this doesn't mean the current code isn't being worked on anymore! We did fix some issues with the current video player code. Notably are changes for Fire TV to direct play more often and a lot of crashes got fixed.
```

Comment:
```suggestion
The new video player design is just the beginning. A project to rewrite the playback code behind it is already in the works. This rewrite should help with the crashes and unnecessary transcoding that happens sometimes. It will also give us the opportunity to add new features like SyncPlay. But this doesn't mean the current code isn't being worked on anymore! We did fix some issues with the current video player code. Notable are changes for Fire TV to direct play more often and a lot of crashes got fixed.
```